src/app/components/header/header.component.scss

```
@@ -0,0 +1,70 @@
+.header{
+    width: 100%;
+    height: 72px;
+    box-sizing: border-box;
+    position: absolute;
+    left: 0%;
+    right: 0%;
+    top: 0%;
+    bottom: 0%;
+    padding-right: 150px;
+    padding-left: 150px;
+    background: #FFFFFF;
+    border: 1px solid #DDE2FF;
+    border-radius: 5px;
+}
+
+.header-content{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 100%;
+}
+.header-logo-label{
```

```suggestion
}

.header-logo-label {
```

Fix this way in other places.

---

Fixed in the resolve commit vaia.